app/lib/common/providers/space_providers.dart

```diff
@@ -21,12 +21,33 @@ final spaceProvider =
   () => AsyncSpaceNotifier(),
 );
 
+final maybeSpaceProvider = AsyncNotifierProvider.autoDispose
```

Comment:
could you add brief word about the purpose of provider ? As its gonna keep increasing so having them commented is better :)